server/src/main/java/org/elasticsearch/cluster/metadata/LifecycleExecutionState.java

```diff
@@ -58,6 +65,7 @@ public record LifecycleExecutionState(
     private static final String SNAPSHOT_INDEX_NAME = "snapshot_index_name";
     private static final String SHRINK_INDEX_NAME = "shrink_index_name";
     private static final String DOWNSAMPLE_INDEX_NAME = "rollup_index_name";
+    private static final String ACTIONS_ORDER_VERSION = "actions_order_version";
```

Comment:
I think we should go with `action_order_version` and avoid the awkward plural-action-but-singular-version noun